Environment_Engine/Modify/TidyPanels.cs

```diff
@@ -0,0 +1,77 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ComponentModel;
+
+using BH.oM.Environment.Elements;
+using BH.oM.Reflection.Attributes;
+
+namespace BH.Engine.Environment
+{
+    public static partial class Modify
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        [Description("Returns a list of Environment Panels with overlapping panels split and merged")] 
+        [Input("panels", "A collection of Environment Panels to tidy")]
+        [Output("panels", "A collection of modified Environment Panels with with overlapping panels split and merged")]
+        public static void TidyPanels(this List<Panel> panels)
```

Comment:
```suggestion
        public static List<Panel> TidyPanels(this List<Panel> panels)
```

Let's make this a compute method instead, this is doing computation more than modification to a single object.